etna/datasets/utils.py

```diff
@@ -187,6 +187,11 @@ def match_target_quantiles(features: Set[str]) -> Set[str]:
     return {i for i in list(features) if pattern.match(i) is not None}
 
 
+def match_target_components(features: Set[str]) -> Set[str]:
```

Comment:
Is it ok to have this function in this task?

Reply:
Actually, this function was scoped in other task which a bit on hold right now, but we need it already.